### DISCUSSION TITLE
Tailwind Compiler in Browser Error:

### DISCUSSION THREAD

**Discussion by ecoinomist** [tailwindlabs/tailwindcss#9717] [Help] [answered]
**Describe your issue**

I'm setting up an in-browser tailwind CSS compiler with post CSS, and tried to reference the play.tailwindcss.com source code. However, I'm struggling to figure out how to fix this error message:

`TypeError: Cannot read properties of undefined (reading 'split')`

**Reproduction URL**

https://stackblitz.com/edit/nextjs-ibyrqs?file=src/tailwind.js

I've setup the `polyfill` folder for Node.js modules in browser, exactly like [modules](https://github.com/tailwindlabs/play.tailwindcss.com/tree/master/src/modules) folder in play repo, and PostCSS compiled successfully with autoprefixer.

But as soon as tailwind is imported, it throws error:

```js
import tailwindcss from 'tailwindcss'; // just importing tailwind causes error
```

Can someone point me to the right direction, please?

--------------------------------------------------------
 
**What version of Tailwind CSS are you using?**

Tailwind v3.1.6

**What build tool (or framework if it abstracts the build tool) are you using?**

"postcss": "^8.4.14", "next": "12.2.3",

**What version of Node.js are you using?**

Node.js v18.4.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Comment by wongjn** [comment, marked as answer]
You're missing out [this line](https://github.com/tailwindlabs/play.tailwindcss.com/blob/351501c0338ebfa3a4f88ae251442cfefa238952/next.config.js#L108) which implements the node package overrides. So for your project, in `next.config.js`:
```diff
 swcMinify: true,
 webpack: (config, { isServer }) => {
+  config.resolve.alias = { ...config.resolve.alias, ...moduleFallback };
   // Polyfill for Tailwind compiler in browser
```

**Reply by ecoinomist** [reply]
Excellent, thank you, it worked. Turns out it had to be overridden as alias, not fallback.
